src/sentry/models/groupopenperiod.py

```diff
@@ -58,11 +58,15 @@ class Meta:
 
         constraints = (
             ExclusionConstraint(
-                name="exclude_overlapping_start_end",
+                name="exclude_overlapping_date_start_end",
                 expressions=[
-                    (models.F("group"), RangeOperators.EQUAL),
+                    ("group", "="),
```

Comment:
Why do we change this to be the string? Can't you just keep it as `modes.F(...)`?

Reply:
hm that was something I had changed when the migration check kept failing, but it shouldn't be necessary